lib/device_vector.cu

```diff
@@ -0,0 +1,73 @@
+#include <device_vector.h>
+#include <tunable_nd.h>
+#include <tunable_reduction.h>
+
+namespace quda
+{
+
+  template <class T> struct AxpbyArg : kernel_param<> {
```

Comment:
The arg and the functor need to be in a separate kernel include file, else this will break on Jitify.

Reply:
https://github.com/lattice/quda/pull/1061/commits/8b75d45d4fd17c8c6e8e368fd783883afcdc4d67